src/vtab/mod.rs

```diff
@@ -472,7 +472,7 @@ impl Values<'_> {
             }
             FromSqlError::OutOfRange(i) => Error::IntegralValueOutOfRange(idx, i),
             #[cfg(feature = "i128_blob")]
-            FromSqlError::InvalidI128Size(_) => Error::InvalidColumnType(idx, value.data_type()),
+            FromSqlError::InvalidI128Size(_) => Error::InvalidColumnType(idx, idx.to_string(), value.data_type()),
```

Comment:
It's dodgy to do `idx.to_string()` here for the name, but I'm not sure it particualrly matters. I almost made the name an Option<String> on the error type, but decided against since it should always be present outside of this case, which should be rare. I'd be willing to change it to do that again if it would be preferred though.